File: java/1014.java
import java.io.IOException;
 import java.util.Scanner;
/**
 * IMPORTANT:
 *      O nome da classe deve ser "Main" para que a sua solução execute
 *      Class name must be "Main" for your solution to execute
 *      El nombre de la clase debe ser "Main" para que su solución ejecutar
 */
public class Main {

    public static void main(String[] args) throws IOException {

        /**
         * Escreva a sua solução aqui
         * Code your solution here
         * Escriba su solución aquí
         */
         Scanner tec = new Scanner(System.in);
        int x = tec.nextInt();
        float y = tec.nextFloat();
        float c = x/y;
        System.out.printf("%.3f km/l\n", c);

    }

}

File: cs/1014.cs
using System;

class URI {

    static void Main(string[] args) {

        /**
         * Escreva a sua solução aqui
         * Code your solution here
         * Escriba su solución aquí
         */
 int x;
            double y, cons;

            x = int.Parse(Console.ReadLine());
            y = double.Parse(Console.ReadLine());

            cons = x / y;

            Console.WriteLine("{0:N3} km/l", cons);
    }

}
